switch/freetype/PKGBUILD

```diff
@@ -1,10 +1,11 @@
 # Maintainer:  Dave Murphy <davem@devkitpro.org>
 # Contributor: cpasjuste <cpasjuste@gmail.com>
 # Contributor: carstene1ns <dev f4ke de>
+# Contributor: Pablo Curiel <pabloacurielz@gmail.com>
 
 pkgname=switch-freetype
-pkgver=2.9
-pkgrel=3
+pkgver=2.10.1
+pkgrel=4
```

Comment:
pkgrel resets to 1 when package version is updated

Reply:
Fixed. Thanks for pointing that out.